string problemPath
string domainPath
---
bool problemResolved
string planStatus
string jsonPath
string jsonObject
